arch/riscv/chunkset_rvv.c

```diff
@@ -0,0 +1,98 @@
+/* chunkset_rvv.c - Genernal version of chunkset
+ * Copyright (C) 2023 SiFive, Inc. All rights reserved.
+ * Contributed by Alex Chiang <alex.chiang@sifive.com>
+ * For conditions of distribution and use, see copyright notice in zlib.h
+ */
+
+#include "zbuild.h"
+
+/*
+ * It's not a optimized implemantation using RISC-V RVV, but a general optimized one.
+ *
+ * RISC-V glibc would enable RVV optimized memcpy at runtime by IFUNC,
+ * so we prefer using large size chunck and copy memory as much as possible.
```

Comment:
Spelling mistake, extra `c` in `chunck`.

Reply:
Thanks, it's done.